src/domain/application/components/summary/InvoicingCustomerSummary.tsx

```diff
@@ -23,40 +27,73 @@ function isInvoicingCustomerEmpty(invoicingCustomer: InvoicingCustomer) {
 
 type Props = {
   invoicingCustomer?: InvoicingCustomer | null;
+  taydennysInvoicingCustomer?: InvoicingCustomer | null;
 };
 
-export default function InvoicingCustomerSummary({ invoicingCustomer }: Readonly<Props>) {
+export default function InvoicingCustomerSummary({
+  invoicingCustomer,
+  taydennysInvoicingCustomer,
+}: Readonly<Props>) {
   const { t } = useTranslation();
 
-  if (!invoicingCustomer || isInvoicingCustomerEmpty(invoicingCustomer)) {
+  if (
+    (!invoicingCustomer || isInvoicingCustomerEmpty(invoicingCustomer)) &&
+    (!taydennysInvoicingCustomer || isInvoicingCustomerEmpty(taydennysInvoicingCustomer))
+  ) {
     return null;
   }
 
   return (
     <>
       <SectionItemTitle>{t('form:yhteystiedot:titles:invoicingCustomerInfo')}</SectionItemTitle>
       <SectionItemContent>
-        <div>
-          <Box marginBottom="var(--spacing-s)">
-            <p>{invoicingCustomer.name}</p>
-            <p>{invoicingCustomer.registryKey}</p>
-          </Box>
-          <Box marginBottom="var(--spacing-s)">
-            <p>{invoicingCustomer.ovt}</p>
-            <p>{invoicingCustomer.invoicingOperator}</p>
-            <p>{invoicingCustomer.customerReference}</p>
-          </Box>
-          <Box marginBottom="var(--spacing-s)">
-            <p>{invoicingCustomer.postalAddress.streetAddress.streetName}</p>
-            <p>
-              {invoicingCustomer.postalAddress.postalCode} {invoicingCustomer?.postalAddress.city}
-            </p>
-          </Box>
+        {invoicingCustomer && (
           <div>
-            <p>{invoicingCustomer.phone}</p>
-            <p>{invoicingCustomer.email}</p>
+            <Box marginBottom="var(--spacing-s)">
+              <p>{invoicingCustomer.name}</p>
+              <p>{invoicingCustomer.registryKey}</p>
+            </Box>
+            <Box marginBottom="var(--spacing-s)">
+              <p>{invoicingCustomer.ovt}</p>
+              <p>{invoicingCustomer.invoicingOperator}</p>
+              <p>{invoicingCustomer.customerReference}</p>
+            </Box>
+            <Box marginBottom="var(--spacing-s)">
+              <p>{invoicingCustomer.postalAddress.streetAddress.streetName}</p>
+              <p>
+                {invoicingCustomer.postalAddress.postalCode} {invoicingCustomer?.postalAddress.city}
+              </p>
+            </Box>
+            <div>
+              <p>{invoicingCustomer.phone}</p>
+              <p>{invoicingCustomer.email}</p>
+            </div>
           </div>
-        </div>
+        )}
+        {taydennysInvoicingCustomer && taydennysInvoicingCustomer !== invoicingCustomer && (
```

Comment:
Täydennys invoicing customer is displayed even if there is no change to invoicing customer info and `muutokset` does not include `invoicingCustomer`:
![Screenshot 2025-01-31 at 9 26 41](https://github.com/user-attachments/assets/2b4946c4-e6dc-4917-89df-130e29c391ad)

Reply:
You're right! I changed this (`InvoicingCustomerSummary`) to be more like `ContactsSummary`.